docs/UserGuide.md

```diff
@@ -12,7 +12,7 @@ It allows users to:
 1. Record all clients in the app
 2. Manage their clients' personal details
 3. Manage their clients' portfolios' details
-4. Easily arrange for meetups with clients
+4. Track your meetings with your clients
```

Comment:
Thanks for the prompt changes to the UG after our meeting with our adviser this morning! I think the updated phrasing of this sentence is definitely better suited to the features of our application.